### ISSUE TITLE
Validator: Ignore warning about Binding for path (.+) has no source

### ISSUE THREAD

**Issue by oliveregger** [ahdis/ch-elm#94]
Binding for path (.+) has no source, so can't be checked

**Comment by oliveregger** [comment]
```json
{    "severity": "information",
    "code": "code-invalid",
    "diagnostics": "Binding for path Bundle.entry[1].resource/*Patient/2c3e0c85-3374-4569-9d6b-6e389049e486*/.address[0].country.extension[0].value.ofType(Coding) has no source, so can't be checked",
    "expression": [ "Bundle.entry[1].resource/*Patient/2c3e0c85-3374-4569-9d6b-6e389049e486*/.address[0].country.extension[0].value.ofType(Coding)" ]
  }
```

**Comment by oliveregger** [comment]
fixed in version 1.3.1
